daemons/pacemakerd/pcmkd_messages.c

```diff
@@ -109,6 +124,35 @@ handle_shutdown_request(pcmk__client_t *c, xmlNode *msg, uint32_t id, uint32_t f
     }
 }
 
+static xmlNode *
+handle_unknown_request(pcmk__request_t *request)
+{
+    const char *op = crm_element_value(request->xml, F_CRM_TASK);
+
+    crm_err("Unknown IPC request %s from %s %s",
+            op, pcmk__request_origin_type(request),
+            pcmk__request_origin(request));
```

Comment:
We don't need this log if we elevate the other one

Reply:
Fixed.